Procfile

```diff
@@ -0,0 +1,2 @@
+smsworker: bundle exec sidekiq -c 2
+emailworker: bundle exec sidekiq -c 2
```

Comment:
Can we add some comments to explain these like you explained them to me a moment ago? Thanks!